client/app/cda/cda.js

```diff
@@ -4,7 +4,7 @@ angular.module('managerApp')
       .state('paas.cda', {
         url: '/cda',
         template: '<div ui-view="cdaDetails"></div>',
-        translations: ['common', 'cda'],
+        translations: ['.'],
```

Comment:
Can't this be automated ?

```js
if (/* I don't know what */) {
	translations = translations ? translations.concat(['.']) : ['.']
}
```

Reply:
technically yes :) it could but :

the syntax ```translations: ['.']``` is already a shorthand to avoid writing : 
```import("./translations/Messages_${translate.use()}.xml").then( /** load trad with angular translate **/ ).then($translate.refresh());```
we wrote this webpack loader to be compatible with existing codebase. On the other hand we don't want to perform too much *magic* behind the scene and prefer the developer to be explicit on what he wants to import. (also having our shared webpack configuration the least complicated and specific is better)

maybe in the future we'll drop the ``` translations: [ ... ] ``` syntax in favor to writing webpack imports by hand i don't know yet :)

if you want some hints on how the current loader works, take a look here : https://github.com/ovh-ux/manager-webpack-config/blob/master/loaders/ui-router-translations.js